cmd/kubectl-kcp-workspace/main.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2021 The KCP Authors.
```

Comment:
nit: 22

Reply:
- Does the year being correct only matter for newly added files? 
- Is there any consequence to it being wrong?

Reply:
Will fix

Reply:
@marun I am not authoritative but my guesses:
- yes
- no